src/components/Navigation/Logout.jsx

```diff
@@ -22,4 +33,13 @@ class Logout extends React.Component {
   }
 }
 
-export default connect()(Logout)
+function mapStateToProps(state) {
```

Comment:
A more es6 way to do this: `mapStateToProps = ({ section = {}, application = {} })`

Then we don't need to have the `|| {}` directly in the function. Of course, these values should be defined in a default state somewhere in the reducers, but that is another issue.

Reply:
Just want to make sure I'm following this, so it would look like this instead?
```
mapStateToProps = ({
  section = {},
  application = {}
})

export default connect(mapStateToProps)(Logout)
```

ESLint complains about that (`Shorthand property assignments are valid only in destructuring patterns`), and would I still need to return something?

Reply:
Sorry, the code I suggested was not all the code you'd need. Expanded:

```
function mapStateToProps({ section = {}, application = {} }) {
  return {
    section,
    app: application
  }
}
```

Reply:
Got it, thanks, updated